### ISSUE TITLE
idea: exam breaks that freeze questions

### ISSUE THREAD

**Issue by trombonekenny** [PrairieLearn/PrairieLearn#8524]
I ran across an online discussions about the [ARE exam](https://www.ncarb.org/pass-the-are/prepare/are-5-0-format) where the (multi-hour) professional exam allows the student to trigger a "break", where they can step away from the exam, get food, restroom, etc. with the caveat that any questions they had started or viewed were then frozen and couldn't be updated when they resumed.

I'm not sure if that's something that would be desirable in PL, or maybe just on long exams, but I didn't want to lose the thought so tracking it here.

**Comment by jonatanschroeder** [comment]
I have had a use-case (outside PL) in the past for something similar. I had a student with accessibility needs that required three times the usual exam time due to severe mobility issues and the need for speech-to-text software. For a 2.5-hour exam this meant the student had almost 8 hours of exam, which was unfeasible to do in a single sitting, so the student had the exam over two days. I had the student in two consecutive terms. The first term I broke the exam into two halves, which didn't work that well, since the student finished one half faster than the other. In the second term I managed to provide the student with an environment similar to the above by breaking down the exam into separate PDFs for each question, and the student could request a PDF from the invigilator one at a time, and had to hand in all completed questions on the first sitting. Having the option above on PL makes a lot of sense from an accessibility point of view for this kind of use case.

**Comment by trombonekenny** [comment]
Similar thought, chemistry profs who moved to PrairieLearn from another platform were accustomed to "hiding" questions after the student answered them. Like, final submission, see the score, then it disappears. I believe their logic was this simplified things for the students to see only what work remained, possibly without dwelling on the previous work, and might have had some heightened exam security by not giving students time to review questions they completed (to tell a friend later).

I guess that idea fits into this thread because it seems to go against traditional test taking norms and transparency to the student, but may have some overall benefits we haven't considered because we haven't used those forms.

**Comment by jonatanschroeder** [comment]
> Similar thought, chemistry profs who moved to PrairieLearn from another platform were accustomed to "hiding" questions after the student answered them. Like, final submission, see the score, then it disappears. I believe their logic was this simplified things for the students to see only what work remained, possibly without dwelling on the previous work, and might have had some heightened exam security by not giving students time to review questions they completed (to tell a friend later).

Just to make sure this is clear, I would distinguish hiding previous questions and freezing them. By freezing I mean not allowing further submissions, but I don't necessarily mean hiding the question itself. I'm not saying this could not be an option, I just want to make sure that this difference is taken into account.

**Comment by trombonekenny** [comment]
> I would distinguish hiding previous questions and freezing them.

Right, I agree. I only brought it up in this thread under the umbrella of non-traditional exam access or views.

**Comment by nwalters512** [comment]
Given that this really is more of a discussion than anything that's immediately actionable by us, I'm going to convert this to a GitHub Discussion.
